src/main/resources/assets/schemas/DatasetSearch.yaml

```diff
@@ -0,0 +1,33 @@
+type: object
+properties:
```

Comment:
There should be a lot more potential properties here (see [DatasetSearchTerm](https://github.com/DataBiosphere/consent/blob/develop/src/main/java/org/broadinstitute/consent/http/models/DatasetSearchTerm.java)). I think we still have a bug where a lot of those fields are not actually being sent to ES right now.